Integrations/integration-IntSights.yml

```diff
@@ -88,6 +89,67 @@ script:
                 return_error('Error in API call to IntSights service %s - [%d] %s' % (path, r.status_code, r.text))
         return r
 
+    def convert_iso_string_to_python_date(date_in_iso_format):
+        iso_format = "%Y-%m-%dT%H:%M:%S"
+        date_in_python_format = datetime.strptime(date_in_iso_format, iso_format)
+        return date_in_python_format
+
+    def convert_python_date_to_unix_millisecond(python_date_object):
+        timestamp_in_unix_millisecond = (python_date_object - datetime(1970, 1, 1)).total_seconds()
+        return timestamp_in_unix_millisecond
+
+    def return_iso_plus_x_days(date_in_iso_format, num_of_days):
+        iso_format = "%Y-%m-%dT%H:%M:%S"
+        date_in_python_format = convert_iso_string_to_python_date(date_in_iso_format)
+        new_date_in_python_format = date_in_python_format + timedelta(days = int(num_of_days))
+        new_date_in_iso_format = new_date_in_python_format.isoformat()
+        return new_date_in_iso_format
+
+    def remove_milliseconds_from_iso(date_in_iso_format):
+        date_parts_arr = date_in_iso_format.split('.')
+        date_in_iso_without_milliseconds = date_parts_arr[0]
+        return date_in_iso_without_milliseconds
+
+    def return_timestamp_plus_x_days(date_in_unix_ms_timestamp, num_of_days):
+        date_in_iso = timestamp_to_datestring(date_in_unix_ms_timestamp)
+        date_in_iso_without_ms = remove_milliseconds_from_iso(date_in_iso)
+        date_in_iso_plus_x_days = return_iso_plus_x_days(date_in_iso_without_ms, num_of_days)
+        timestamp_in_unix_ms_plus_x_days = date_to_timestamp(date_in_iso_plus_x_days)
+        return timestamp_in_unix_ms_plus_x_days
+
+    def update_params_if_end_date_given(params, time_delta_in_days_int):
```

Comment:
Not sure if i want to support that, you will need to write some good documentation for that if we are keeping it